n2y/rich_text.py

```diff
@@ -42,12 +42,14 @@ def to_markdown(self):
 
     def plain_text_to_pandoc(self):
         ast = []
-        match = re.findall(r"( +)|(\S+)|(\n+)|(\t+)", self.plain_text)
+        match = re.findall(r"( +)|(\xa0+)|(\S+)|(\n+)|(\t+)", self.plain_text)
 
         for m in match:
-            space, word, newline, tab = m
+            space, latin1_space, word, newline, tab = m
```

Comment:
Are there any other ways to encode a space?

Reply:
Good question, I don't think so. I looked through the Latin1 character set (which looks to be what the Notion API supports) and only found two ways to encode a space [1]. There is the default encoding and then this new one (also known as a "non-breaking space"). 

[1]: https://kb.iu.edu/d/aepu